css/audio.css

```diff
@@ -284,7 +284,7 @@ hr {
   margin: 15px;
   padding-left: 27px;
   padding-right: 27px;
-  padding-bottom: 12px
+  padding-bottom: 12px;

```

Comment:
For using Padding, there is no need to write like this.

```diff 
- padding-left: 27px;
- padding-right: 27px;
- padding-bottom: 12px;
+ padding: 12px 27px;
```

> **Note**
> padding: `top` `right` `bottom` `left`;

You can search for more information.